plans/plan.md

```diff
@@ -0,0 +1,26 @@
+# Plans for TRBoom's Personal Website
+
+## Tutorial for a Fake News Detector Powered by Transformers
+
+
+### Overiew
+I'm building a fake news detector for another class using a transformer based approach. The website would chronical my approaches or be a tutorial on how to get a transformer up and running.
```

Comment:
Topic sounds very interesting, although, it would be appropriate to include a bit of background here to introduce the unfamiliar reader to the topic.

You sound uncertain of the main goal of the website. Will you be  illustrating your approaches, offering tutorials, or both?